packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -16,32 +16,32 @@ For information about the element that opens the dropdown container, read the ar
 - **Label:** Set a label for the dropdown item. We typically use short labels including verbs.
-- **Anchor:** An anchor defines where the dropdown is opened. If no anchor is defined the trigger element is used as the anchor.
-- **Trigger:** The trigger defines which element opens the dropdown. A trigger should also be defined for a dropdown submenu. Typically the Button is the trigger element.
+- **Anchor:** An anchor defines where the dropdown is opened. When no anchor is defined the trigger element is used as the anchor.
+- **Trigger:** The trigger defines which element opens the dropdown. A trigger should also be defined for a dropdown submenu. We typically use a button as the trigger element.
 - **Close Behavior:** Defines which click action closes the dropdown. A dropdown of a submenu is closed with the higher-level dropdown, regardless of its own closing behavior.
 Three Options are possible: 
     - Inside: clicking within the dropdown closes the dropdown.
     - Outside: clicking outside the dropdown closes the dropdown.
-    - Both: clicking within and outside of the dropdown closes the dropdown.
-- **Placement:** Defines where the dropdown appears when it is opened. The position depends on the element that opens the drop-down list (e.g. button). By default, the dropdown is displayed at the bottom right of the button. 
-You can also choose between different directions (top, bottom, left, right) and two options for the alignment on the button (start, end). If there is not enough space for the selected placement, it will be corrected automatically. The placement of the submenu is always generated automatically. We typically use the default placement option to ensure consistency. 
+    - Both: clicking within and outside the dropdown closes the dropdown.
+- **Placement:** Defines where the dropdown appears when it is opened. The position depends on the element that opens the dropdown (e.g. button). By default, the dropdown is displayed at the bottom right of the button. 
+You can choose between different directions (top, bottom, left, right) and two options for the alignment on the button (start, end). When there is not enough space for the selected placement, it will be corrected automatically. The placement of the submenu is always generated automatically. We typically use the default placement option to ensure consistency. 
 - **Date Selection:** Use the element [Date dropdown](date-dropdown.md) to get a date selection in the dropdown.
 
 ![Dropdown Examples](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2372-2696&mode=design&t=OVHeXvLZYLkP2CzN-4)
 
 ## Behavior in context
 - **Text truncation:**  Item and header labels are not truncated. The whole text in the items and headers consists of one line only. The text is cut off after 58 characters with "...".
```

Comment:
maybe it's the time - "..." is called a truncation, isn't it?

Reply:
Here I wanted to clarify where exactly it cuts off and how exactly. Since there is also the variant with (...).